WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostActivity.java

```diff
@@ -1113,6 +1113,11 @@ public void updatePostContent(boolean isAutoSave) {
      */
 
     private void fetchMedia(Uri mediaUri) {
+        if (mediaUri == null) {
```

Comment:
What are the chances that there may be multiple null media URLs? I'm thinking we want to avoid barraging the user with toasts if there are multiple failures.